src/aosm/azext_aosm/inputs/helm_chart_input.py

```diff
@@ -124,9 +132,56 @@ def from_chart_path(
                 artifact_name=name,
                 artifact_version=version,
                 chart_path=chart_path,
-                default_config=default_config,
+                default_config_path=default_config_path,
             )
 
+    def validate_template(self) -> None:
+        """
+        Perform validation on the Helm chart template by running `helm template` command.
+
+        :return: Error message if the `helm template` command fails.
+        """
+        logger.debug("Performing validation on Helm chart %s.", self.artifact_name)
+
+        check_tool_installed("helm")
+
+        if self.default_config_path:
+            cmd = [
+                "helm",
+                "template",
+                self.artifact_name,
+                self.chart_path,
+                "--values",
+                self.default_config_path,
+            ]
+        else:
+            cmd = [
+                "helm",
+                "template",
+                self.artifact_name,
+                self.chart_path,
+            ]
+
+        try:
+            result = subprocess.run(cmd, capture_output=True, check=True)
+            helm_template_output = result.stdout
+            self.helm_template.write(helm_template_output)
+
+            logger.debug(
+                "Helm template output for Helm chart %s:\n%s",
+                self.artifact_name,
+                helm_template_output,
+            )
+            return ""
+        except subprocess.CalledProcessError as error:
+            # Return the error message without raising an error.
+            # The errors are going to be collected into a file by the caller of this function.
+            error_message = error.stderr.decode()
+            error_message = error_message.replace(
+                "\nUse --debug flag to render out invalid YAML", ""
```

Comment:
Is there a reason not to run the command with `--debug` ourselves and provide this useful additional info? 

If there is a good reason (e.g. too verbose) could you add something to the error template along the lines of "For more detailed information on each error, run `helm template <chart name> --debug` (or whatever the command is) yourself from the command line.

Reply:
I should add a comment about this not being too useful